imports/ui/lists/trendingAgents.jade

```diff
@@ -20,18 +20,28 @@ template(name="trendingAgents")
 
     ul.trending-agents.list--sub-group.list-unstyled
       each trendingAgents
-        li(data-agentname=word)
-          a.fmia-word(href="/detail/#{word}")= word
-          .relative-strength
-            +powerBars
+        li
+          +trendingAgent
       else
         if ready
           p.no-priors.centered.subtle No Trending Infectious Agents in the last
             | {{trendingRange}}
         else
           +loader classes='inline'
 
-template(name="powerBars")
-  .power-bars(data-toggle="tooltip" data-placement="left" title="The number of bars indicates the relative strength of the trend.")
-    each bars
-      .power-bar
+template(name="trendingAgent")
+  a.fmia-word(href="/detail/#{word}")= word
+  if seasonal
+    a.fa.fa-sun-o.seasonal(
+      data-toggle="tooltip"
+      data-placement="right"
+      title="This may be a seasonal trend."
+    )
```

Comment:
We may want to not use a anchor tag here since it may cause some confusion on the user's end (the color change on hover) — they may expect an action in addition to the showing the tooltip.